packages/yoga/src/Text/native/Text.jsx

```diff
@@ -74,6 +74,7 @@ Text.displayName = 'Text';
 
 Text.propTypes = {
   inverted: bool,
+  fontSize: oneOfType([string, number]),
```

Comment:
Maybe we should add the accepted values for this prop, just like we did on `size` prop. My suggestion:

```suggestion
  fontSize: oneOfType([
      'xxsmall',
      'xsmall',
      'small',
      'medium',
      'large',
      'xlarge',
      'xxlarge',
      'xxxlarge',
      'huge',
    ]),
```

What do you think?

Reply:
Hi, @caiotracera! First I would like to thank you for the review, I am very happy for your attention. 🚀🎉

Interesting! On second thought, it really makes sense to add the values accepted by the prop.